go.mod

```diff
@@ -204,4 +204,7 @@ replace google.golang.org/protobuf => github.com/lightninglabs/protobuf-go-hex-d
 
 replace github.com/lightninglabs/loop/swapserverrpc => ./swapserverrpc
 
+// Temporary added until https://github.com/lightninglabs/aperture/pull/135 is merged.
+replace github.com/lightninglabs/aperture => github.com/starius/aperture v0.3.2-rc1-beta
```

Comment:
Reminder to not merge this PR until the dependent PR was merged.

Reply:
Fixed